recipes/default.rb

```diff
@@ -19,8 +19,10 @@
 
 influxdb_config node['influxdb']['config_file_path'] do
   config node['influxdb']['config']
+  notifies :restart, 'service[influxdb]', :delayed
 end
 
 service 'influxdb' do
-  action [:enable, :start]
+  supports :restart => true
```

Comment:
@nilroy because Rubocop enforces a certain style of Ruby. `restart: true` and `:restart => true` are equivalent in the interpreter's eyes but some people like the style of the former better.